framework/helpers/BaseStringHelper.php

```diff
@@ -338,4 +338,36 @@ public static function base64UrlDecode($input)
     {
         return base64_decode(strtr($input, '-_', '+/'));
     }
+
+    /**
+     * Masks a token to make it uncompressible.
+     * Applies random mask to a token and prepends mask used to the result making the string always unique.
+     * Used to mitigate BREACH attack by randomizing how token is outputted on each request.
```

Comment:
Mitigate is part of the vocabulary. Regardless, please make a final decision on wording before I change this 10 times ;-)